src/test/java/automationpractice/com/pages/ProductDetailsPage.java

```diff
@@ -0,0 +1,251 @@
+    private WebElementFacade clickProceedToCheckoutButton;
+
+    @FindBy(className = "icon-plus")
+    private WebElementFacade clickPlusQuantityButtonOnProductDetailsPage;
+
+    @FindBy(className = "icon-minus")
+    private WebElementFacade clickMinusQuantityButtonOnProductDetailsPage;
+
+    @FindBy(xpath = "//*[@id='group_1']//option[1]")
+    private WebElementFacade selectSSize;
+
+    @FindBy(xpath = "//*[@id='group_1']//option[2]")
+    private WebElementFacade selectMSize;
+
+    @FindBy(xpath = "//*[@id='group_1']//option[3]")
+    private WebElementFacade selectLSize;
+
+    @FindBy(id = "color_14")
+    private WebElementFacade changeColorToBlue;
+
+    @FindBy(id = "color_13")
+    private WebElementFacade changeColorToOrange;
+
+    @FindBy(id = "color_8")
+    private WebElementFacade changeColorToWhite;
+
+    @FindBy(id = "color_11")
+    private WebElementFacade changeColorToBlack;
+
+    @FindBy(id = "color_15")
+    private WebElementFacade changeColorToGreen;
+
+    @FindBy(id = "color_16")
+    private WebElementFacade changeColorToYellow;
+
+    @FindBy(className = "btn-twitter")
+    private WebElementFacade clickOnTwitterButton;
+
+    @FindBy(className = "btn-facebook")
+    private WebElementFacade clickOnFacebookButton;
+
+    @FindBy(className = "btn-google-plus")
+    private WebElementFacade clickOnGooglePlusButton;
+
+    @FindBy(className = "btn-pinterest")
+    private WebElementFacade clickOnPinteresButton;
+
+    private int currentQuantityInProductDetailsPage;
+
+    private String currentSizeInProductDetailsPage;
+
+    @FindBy(xpath = "//*[@id = 'color_to_pick_list']//*[contains(@id,'color')]")
+    private List<WebElementFacade> listOfColorsAndSizes;
+
+    private String currentColorInProductDetailsPage;
+
+    public ProductDetailsPage() {
+    }
+
+    public void clickOnAddToCartButton() {
+        clickOnAddToCartButton.click();
+    }
+
+    public void clickProceedToCheckoutButton() {
+        clickProceedToCheckoutButton.click();
+    }
+
+    private void clickPlusQuantityButtonOnProductDetailsPage() {
+        clickPlusQuantityButtonOnProductDetailsPage.click();
+    }
+
+    private void clickMinusQuantityButtonOnProductDetailsPage() {
+        clickMinusQuantityButtonOnProductDetailsPage.click();
+    }
+
+    public int clickPlusAndMinusButtonGivenNumberOfTimesOnProductDetailsPage(int plus, int minus) {
+        currentQuantityInProductDetailsPage = 1;
+        for (int i = 0; i < plus; i++) {
+            waitABit(1000);
+            clickPlusQuantityButtonOnProductDetailsPage();
+            currentQuantityInProductDetailsPage++;
+        }
+        for (int i = 0; i < minus; i++) {
+            waitABit(1000);
+            clickMinusQuantityButtonOnProductDetailsPage();
+            currentQuantityInProductDetailsPage--;
+        }
+        return currentQuantityInProductDetailsPage;
+    }
```

Comment:
Nie używajmy waitABit jeśli to nie jest naprawdę potrzebne.
Ta metoda powinna być w jakiejś klasie Steps
Nie rozumiem do końca po co zwracasz tu int a nie void